examples/draft-0-10-0/playground/src/App.css

```diff
@@ -140,3 +156,14 @@
   /** sad :( but seems to not be a way to overwrite that by configuration */
   background: var(--main-background) !important;
 }
+
+.RichEditor-editor {
+  padding: 10px 20px;
+}
+
+/**
+ * basic style for handling nested lists
+ */
+.RichEditor-editor li[data-block] li[data-block] {
+  margin-left: 20px;
```

Comment:
I'm surprised this doesn't have some default styling that does the indentation. Oh well.

Reply:
we don't have nested lists support, we use CSS for the depth :(